frontend/javascripts/oxalis/view/right-menu/meshes_view.js

```diff
@@ -130,6 +143,9 @@ class MeshesView extends React.Component<Props, { currentlyEditedMesh: ?MeshMeta
               Import
             </ButtonComponent>
           </Upload>
+          <ButtonComponent icon="download" onClick={this.props.downloadIsosurface}>
```

Comment:
Should this include a note that only the isosurface of the currently active cell will be downloaded?